backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -126,12 +126,12 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   def preProcessWomFile(womFile: WomFile): WomFile = womFile
   
   /** @see [[Command.instantiate]] */
-  final lazy val commandLinePreProcessor: WomEvaluatedCallInputs => Try[WomEvaluatedCallInputs] = {
-    inputs =>
-      TryUtil.sequenceMap(inputs mapValues WomFileMapper.mapWomFiles(preProcessWomFile, inputsToNotLocalize)).
-        recoverWith {
-          case e => Failure(new IOException(e.getMessage) with CromwellFatalExceptionMarker)
-        }
+  final def commandLinePreProcessor(inputs: WomEvaluatedCallInputs): Try[WomEvaluatedCallInputs] = {
+    val map = inputs map { case (k, v) => k -> WomFileMapper.mapWomFiles(preProcessWomFile, inputsToNotLocalize)(v) }
+    TryUtil.sequenceMap(map).
```

Comment:
Could `sequenceMap` become a `traverse` if ` WomFileMapper.mapWomFiles` makes `ErrorOr`s?

Reply:
I think so but that won't be happening in this PR 😛

Reply:
😢